web/src/features/panels/zone/ZoneHeaderTitle.tsx

```diff
@@ -44,19 +44,24 @@ export default function ZoneHeaderTitle({
                 size={18}
                 className="shadow-[0_0px_3px_rgba(0,0,0,0.2)]"
               />
-              <div className="ml-2 flex flex-row">
-                <h2
-                  className="max-w-[300px] overflow-hidden truncate text-lg font-medium sm:max-w-[230px] md:max-w-[270px]"
-                  data-test-id="zone-name"
-                >
-                  {title}
-                </h2>
-                {isSubZone && (
-                  <p className="ml-2 flex w-auto items-center whitespace-nowrap rounded-full bg-gray-200 py-0.5 px-2  text-sm dark:bg-gray-900">
-                    {countryName || zoneId}
-                  </p>
-                )}
-              </div>
+              <TooltipWrapper
+                tooltipContent={title.length > 20 ? title : undefined}
+                side="bottom"
+              >
+                <div className="ml-2 flex flex-row">
+                  <h2
+                    className="max-w-[300px] overflow-hidden truncate text-lg font-medium sm:max-w-[230px] md:max-w-[270px]"
```

Comment:
I think it makes sense to increase the max width here to avoid multiline names as much as possible. Making it wider would also avoid making it higher when the name wraps and should look more like you would expect it to. Maybe we can even skip the max width entirely for large screens?

Reply:
This was just an indenting change. This PR is just about adding the tooltip!

We should improve the header width scaling at some point though, I notice it playing up quite a bit on Arc with the narrower window space.

Reply:
Oh 😅

Then I'm sorry for dumping all this feedback on a somewhat unrelated PR but we should adress it at some point.